wg-meetings/2020-03-25.md

```diff
@@ -0,0 +1,93 @@
+* Peter: how should we handle it? Discuss next meeting?
+* Mattheus: yes discussing on next meeting + opening a PR to add to empty files
+* Action items: reach out to website repo [Peter], collaborator summit submissions (will write a draft and discuss with WG) [Gireesh], Opening a PR about native addon use-case and gap [Matheus]
+
+* Diagnostics "Best Practices" Guide? [#211](https://github.com/nodejs/diagnostics/issues/211)
+* As part of user journeys we also provide best practices we feel we can collapse the two and remove this from the agenda
+* Worth bringing it back if user journey outcomes doesn’t make it to the docs (PR’s are not open or merged)
+
+* \[async_hooks\] stable API - tracking issue [#124](https://github.com/nodejs/diagnostics/issues/124)
+* A benchmark was added by @legendecas
+* Promises are slow, currently trying to tackle it
+* Is Promises the last missing piece?
+* Not sure, but maybe API still needs some work
+* Gireesh: async local storage API I think will require refining
+* Gireesh: async hooks and async local storage would go hand in hand? Like would they come out together from experimental? I would expect async local storage is based on async-hooks
+* Gireesh: async-hooks looks like making a good progress to come out from experimental
+* Stephen probably has more context about it
+* What would be expected from this WG or is it informal at this point?
+* Gireesh: scoping is probably APIs and documentation
+* Peter: we could make a deep dive on how async-hooks affect our current tooling and what new tooling it will make possible
+* Things to discuss on deep dive:
+  * docs, API, current tools, new tools, async local storage
+* Action item: opening an issue with deep dive proposal and proposed time slots [Peter]
+
+* proposal: elevate diagnostic report to tier1
+[369]https://github.com/nodejs/diagnostics/issues/369
+  * It passes all the requirements to become Tier 1
+  * Peter: Do we have a process / voting system to make this happen?
+  * Gireesh: it’s majority based usually, maybe open for X days
+  * Matheus: we don’t have any processes just follow the normal Node.js guidelines
+  * Matheus: there was some discussion to move one tier at time, but not sure it was applied
+  * Peter: how about keeping it open for one week in diagnostics repo than opening the PR in the main repo
+  * Gireesh, Mattheus: makes sense
+  * Matheus: we don’t really enforce tiers today so it’s more a formal think
+  * Matheus: we should start to discuss tier supports with release WG
+  * Matheus: today if a tier 1 tool breaks, it doesn’t block release process
+  * Gireesh: https://github.com/nodejs/node/blob/master/doc/guides/diagnostic-tooling-support-tiers.md#diagnostic-tooling-support-tiers says it should be blocked, if it’s not enforced we need to either change the docs to draft or accept it
+  * Action item: open issue with discussion about tier in release wg [Matheus]
```

Comment:
FWIW had the action item been opened after the Diagnostics meeting on 25th we could have discussed it in the Release WG meeting the next day on the 26th.

Reply:
@mmarchini Are you still planning to open an issue in the Release WG? If it needs discussion by the Release WG we can add it to this week's meeting: https://github.com/nodejs/Release/issues/564

Reply:
I'll try to open one tomorrow. No need to discuss in this week Release meeting though.